<ui version="4.0" >
 <class>udg::QInputOutputPacsWidgetBase</class>
 <widget class="QWidget" name="udg::QInputOutputPacsWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="udg::QStudyTreeWidget" native="1" name="m_studyTreeWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="topMargin" >
      <number>6</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_viewButton" >
       <property name="text" >
        <string>View</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/view.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_retrieveButton" >
       <property name="text" >
        <string>Retrieve</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/retrieve.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>371</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="m_queryInProgressLabel" >
           <property name="lineWidth" >
            <number>1</number>
           </property>
           <property name="midLineWidth" >
            <number>0</number>
           </property>
           <property name="text" >
            <string>Query in progress</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="m_queryAnimationLabel" >
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_cancelQueryButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Cancel query</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/cancel.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QStudyTreeWidget</class>
   <extends>QWidget</extends>
   <header>qstudytreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections/>
</ui>
